src/iop/grain.c

```diff
@@ -530,6 +628,14 @@ void gui_init(struct dt_iop_module_t *self)
   gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(g->scale2), TRUE, TRUE, 0);
   gtk_widget_set_tooltip_text(g->scale2, _("the strength of applied grain"));
   g_signal_connect(G_OBJECT(g->scale2), "value-changed", G_CALLBACK(strength_callback), self);
+
+  /* midtones bias */
+  g->scale3 = dt_bauhaus_slider_new_with_range(self, 0.0, 100.0, 1.0, p->midtones_bias * 100, 2);
+  dt_bauhaus_widget_set_label(g->scale3, NULL, _("midtones bias"));
+  dt_bauhaus_slider_set_format(g->scale3, "%.0f%%");
+  gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(g->scale3), TRUE, TRUE, 0);
+  gtk_widget_set_tooltip_text(g->scale3, _("amount of midtones bias from the photographic paper response modeling"));
```

Comment:
Maybe add a sentence about what low/high values do. Not critical, but who reads the manual … 😄